exporting-grafana-snapshots.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Exporting Grafana Snapshots
+summary: How to export snapshots of Grafana Dashboard, and how to visualize these files
+aliases: ['/docs/dev/exporting-grafana-snapshots']
```

Comment:
```suggestion
```
`aliases` is for old docs to be deleted or moved. We do not need to add `aliases` for new docs.